python/lib/cloudutils/serviceConfigServer.py

```diff
@@ -107,7 +107,7 @@ def checkHostName():
         bash("chown cloud.cloud /var/run/cloudstack-management.pid")
         #distro like sl 6.1 needs this folder, or tomcat6 failed to start
```

Comment:
@borisroman Did you see this remark in the code? Is that about the line you remove here? I have no clue what `sl 6.1` is.. Seems like a RHEL like release? Scientific Linux? I dunno! Does it hurt to have the dir? We better remove it later when master is unfrozen IMHO as we do not have a change to test this properly otherwise.